docker/configuration.docker.py

```diff
@@ -77,3 +77,9 @@
 SHORT_TIME_FORMAT = os.environ.get('SHORT_TIME_FORMAT', 'H:i:s')
 DATETIME_FORMAT = os.environ.get('DATETIME_FORMAT', 'N j, Y g:i a')
 SHORT_DATETIME_FORMAT = os.environ.get('SHORT_DATETIME_FORMAT', 'Y-m-d H:i')
+
+# Enforcement of unique IP space can be toggled on a per-VRF basis.
+# To enforce unique IP space within the global table (all prefixes and IP addresses not assigned to a VRF),
+# set ENFORCE_GLOBAL_UNIQUE to True.
+ENFORCE_GLOBAL_UNIQUE = os.environ.get('ENFORCE_GLOBAL_UNIQUE', '')
```

Comment:
If this is a boolean, shouldn't the default be `false` instead of an empty string?

Reply:
My intention was to use the default set here if it was undefined in the env https://github.com/digitalocean/netbox/blob/252be84bf07a27868c9e5548558834b2fd62e3d8/netbox/netbox/settings.py#L44

Hence there should be no default.
I fix!